src/core/symbology-ng/qgssymbollayerutils.h

```diff
@@ -332,6 +332,14 @@ class CORE_EXPORT QgsSymbolLayerUtils
     static void createDisplacementElement( QDomDocument &doc, QDomElement &element, QPointF offset );
     static bool displacementFromSldElement( QDomElement &element, QPointF &offset );
 
+    /**
+     * @brief Creates a SE 1.1 anchor point element as a child of the specified element
```

Comment:
Argh - sorry, I should have picked this up on the earlier reviews, but this should be 

\brief \param \note \returns instead of @. Then sipify can auto translate these to correct python docs.